src/main/paradox/docs/releases/v1.3-release-notes.md

```diff
@@ -1,16 +1,70 @@
 # v1.3 Release Notes
 
-@@@ note
+This is an incremental software release that adds several improvements across the board. The API is backwards
+compatible to v1.2 (and the rest of the v1.x releases, i.e. v1.1 and v1.0) but requires rebuilding all indices. Please
+visit the @ref:[migration instructions](v1.2-to-v1.3-migration.md) for the exact steps necessary to upgrade from the
+previous release.
 
-This release is under development.
+## Highlights
 
-@@@
+The goal of this release is to expand on the benefits of using a fully managed data and knowledge graph system with
+new capabilities around data search, transformation and exchange.
```

Comment:
new capabilities around data aggregation, transformation and presentation ?

Some mention about federation?

Reply:
that that's "exchange" relates to... federation is mentioned in greater detail below.